web/src/settings_invites.ts

```diff
@@ -164,9 +165,10 @@ function do_revoke_invite({
             }),
             $("#home-error"),
         );
+        dialog_widget.hide_dialog_spinner();
     }
 
-    $revoke_button.prop("disabled", true).text($t({defaultMessage: "Working…"}));
+    $revoke_button.prop("disabled", true);
```

Comment:
Don't think there is a need to disable this button. We should just disable the button when the invite is sent successfully to retain the original behavior.